nuget/runners/nunit.console-runner-with-extensions.nuspec

```diff
@@ -26,14 +26,14 @@
     <releaseNotes>This release uses the latest version of the TeamCityEventListener extension.</releaseNotes>
     <language>en-US</language>
     <tags>nunit test testing tdd runner</tags>
-    <copyright>Copyright (c) 2016 Charlie Poole</copyright>
+    <copyright>Copyright (c) 2017 Charlie Poole</copyright>
     <dependencies>
       <group>
-        <dependency id="NUnit.ConsoleRunner" version="$version$" />
-        <dependency id="NUnit.Extension.NUnitProjectLoader" version="$version$" />
-        <dependency id="NUnit.Extension.VSProjectLoader" version="$version$" />
-        <dependency id="NUnit.Extension.NUnitV2ResultWriter" version="$version$" />
-        <dependency id="NUnit.Extension.NUnitV2Driver" version="$version$" />
+          <dependency id="NUnit.ConsoleRunner" version="3.5.0" />
```

Comment:
`$version$` allows you to specify a version at the NuGet command line, doesn't it? Is there maybe a script that needs this?